examples/cors/cors_middleware.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import "github.com/savsgio/atreugo/v10"
+
+// CORS Setting
+var (
```

Comment:
This configuration should be a config struct that is passed to a `NewCorsMiddleware(config)`  function to build the middleware with custom options

Reply:
OK, sorry , I didn't check that function.

Reply:
I see, CORS to be a atregugo core middlware not only in example.

Reply:
Yep